lib/cog/command/pipeline/executor.ex

```diff
@@ -412,8 +416,8 @@ defmodule Cog.Command.Pipeline.Executor do
     end
```

Comment:
I think you may be right. When this function was originally written, our rendering logic was a bit different; now we always pass the rendered `message` in, so the original reason for function has kind of gone away. I can do a follow-on PR to simplify / make it disappear.